### ISSUE TITLE
Where are the Haxe communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/haxe#39]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Haxe supported by one or more large organizations?
* Does Haxe have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Haxe? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Haxe? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Haxe? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Haxe?
* Are there popular newsletters for Haxe?
* Is Haxe taught at programming bootcamps? (If so, what are some examples?)
* Is Haxe taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Haxe?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by al-arz** [comment]
Perhaps @pchertok would like to answer these. At least he seems to be the right person to do so.

**Comment by ErikSchierboom** [comment]
@Nezteb Whether to work on this issue depends on you :) You could look for the above suggestions to possibly help find other maintainers. You could also for now ignore it and return to it later.

**Comment by davidbruce** [comment]
Once this repo is in a bit more organized I will be reaching out to the Haxe community for assistance via:
- [Official Forum](https://community.haxe.org/)
- [Gitter](https://gitter.im/HaxeFoundation/haxe)
- [Haxe Roundup](https://haxe.io/)
